lms/djangoapps/course_api/views.py

```diff
@@ -356,6 +359,12 @@ def get_queryset(self):
             mobile_search=form.cleaned_data.get('mobile_search', False),
         )
 
+    def post(self, request, *args, **kwargs):
```

Comment:
could you add a docstring here so that the generated OpenAPI/Swagger will  have an explanation?

Reply:
Hi, @deborahgu, it's the same as the GET endpoint. I've added the POST example to the class docstring, is that good enough?

Reply:
the class docstring would be fine but I don't see it there -- am I missing it? Just explaining what should be in the payload would be good.

Reply:
Sorry @deborahgu I missed pushing the changes. However, I found an issue with the implementation and will have to re-work it. I'll mark this as a draft for the time being.